holoviews/element/path.py

```diff
@@ -193,22 +205,36 @@ class Ellipse(BaseShape):
 
     height = param.Number(default=1, doc="The height of the ellipse.")
 
-    aspect= param.Number(default=1.0, doc="The aspect ratio of the ellipse.")
+    width = param.Number(default=1, doc="The width of the ellipse.")
+
+    aspect= param.Number(default=1.0, doc="""
+       Final divisor only applied to the final height.""")
 
```

Comment:
I think aspect is very useful.  In some situations one wants to control height and width explicitly (e.g. to enclose a certain area), and in others one wants to control the shape and size separately (with aspect setting the shape, and height setting the size).  Neither case is more valid than the other.